test/components/generators/chat/test_hugging_face_local.py

```diff
@@ -78,6 +79,15 @@ def custom_tool_parser(text: str) -> Optional[List[ToolCall]]:
 
 
 class TestHuggingFaceLocalChatGenerator:
+    @pytest.fixture
+    def mock_model_info(self, monkeypatch):
+        """Mock the model_info function to prevent real HTTP requests."""
+
+        def mock_model_info(*args, **kwargs):
+            return {"pipeline_tag": "text2text-generation"}
+
+        monkeypatch.setattr("huggingface_hub.HfApi.model_info", mock_model_info)
```

Comment:
Can we use this fixture?
https://github.com/deepset-ai/haystack/blob/63781afd8f0df862588f7597de04956f947e303a/test/components/generators/chat/test_hugging_face_local.py#L39-L40

Reply:
done